universal-application-tool-0.0.1/app/services/export/ExporterService.java

```diff
@@ -94,8 +94,7 @@ CsvExportConfig generateDefaultCsvConfig(long programId) {
 
     // Create a map from a key <block id, question index> to an answer with every application. It
     // doesn't matter which answer ends up in the map, as long as every <block id, question index>
-    // is
-    // accounted for.
+    // is accounted for.
```

Comment:
praise: thanks for fixing this formatting!

Reply:
thanks!